src/gui/src/mainWindow.cpp

```diff
@@ -588,6 +588,7 @@ void MainWindow::createActions()
   connect(exit_, &QAction::triggered, this, &MainWindow::exit);
   connect(this, &MainWindow::exit, viewers_, &LayoutTabs::exit);
   connect(fit_, &QAction::triggered, viewers_, &LayoutTabs::fit);
+  connect(fit_, &QAction::triggered, clock_viewer_, &ClockWidget::fit);
```

Comment:
Thanks for looking into this.

Fit in the main window and fit in the clock viewer should be separate.  You might well want to fit one without fitting the other.   You should create an independent action for the ClockWidget.

Reply:
When you say independent action for the ClockWidget, do you want a different hot key for the fit for clock viewer?  If so, what key should I use?  If not, please let me know how do we decide when user press 'f', whether it goes to the layout window or the clock tree viewer?  Thanks.